### ISSUE TITLE
GCSFS doesn't specify minimum version on install, but loose version crashes

### ISSUE THREAD

**Issue by illusional** [intake/filesystem_spec#661]
Noting that the setup.py only sets that the package has to be installed:

https://github.com/intake/filesystem_spec/blob/61ec18fb0a77d3a524a6b2d72340af3748c5bf2a/setup.py#L46

But when using it, the registry will throw an Exception due to:
https://github.com/intake/filesystem_spec/blob/61ec18fb0a77d3a524a6b2d72340af3748c5bf2a/fsspec/registry.py#L190

Exception
```stack
  File "/opt/conda/default/lib/python3.6/site-packages/fsspec/core.py", line 438, in open
    **kwargs,
  File "/opt/conda/default/lib/python3.6/site-packages/fsspec/core.py", line 288, in open_files
    expand=expand,
  File "/opt/conda/default/lib/python3.6/site-packages/fsspec/core.py", line 599, in get_fs_token_paths
    cls = get_filesystem_class(protocol)
  File "/opt/conda/default/lib/python3.6/site-packages/fsspec/registry.py", line 213, in get_filesystem_class
    register_implementation(protocol, _import_class(bit["class"]))
  File "/opt/conda/default/lib/python3.6/site-packages/fsspec/registry.py", line 234, in _import_class
    "higher is required".format(mod.__name__, version, minversion)
RuntimeError: 'gcsfs=0.2.2' is installed, but version '0.3.0' or higher is required
```

I can open a PR to bump this version, but wanted to check what "LooseVersion" means before I did that.

**Comment by martindurant** [comment]
As of this year, fsspec and gcsfs have exact version pinning, so this line is not useful anymore. However, there is no reasonable way to go back and enforce versioning on old releases, I'm afraid you'll have to manually make sure that your fsspec and gcsfs are compatible.

**Comment by martindurant** [comment]
Fixed in #662
